src/pages/ContentfulPage.vue

```diff
@@ -12,6 +12,7 @@
 				:content="content"
 				v-bind="getComponentOptions(content.key)"
 				:class="wrapperClass"
+				data-section-type="contentful-section"
```

Comment:
Open to other names/selectors here, just needed a way to queryselector all of the contentful sections.